appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/spotlight/interpolator/SpotlightSlider.kt

```diff
@@ -62,9 +64,24 @@ class SpotlightSlider<NavTarget : Any>(
         val zIndex: Float = 1f,
         val aspectRatio: Float = 0.42f,
         val rotation: Float = 0f,
-        override val isVisible: Boolean = true
+        val scrollValue: () -> Float,
+        private val width: Dp,
+        private val activeWindow: Float
     ) : Interpolatable<Props>, HasModifier, Animatable<Props>, BaseProps {
 
+        private val activeWindowOffset = (activeWindow * width.value).dp
+
+        override val isVisible: Boolean
+            get() {
+                val currentOffset = (scrollValue() * width.value).dp
```

Comment:
Please check against my changes, the scroll offset should already be added as a displacement, so you don't need to add it a second time.